bin/cli.js

```diff
@@ -99,6 +99,8 @@ function initKnex(env, opts) {
 
   const resolvedConfig = resolveEnvironmentConfig(opts, env.configuration);
   const knex = require(env.modulePath);
+  // attach working config after it is populated
+  env.workingConfig = resolvedConfig;
```

Comment:
Do we need yet another config on top of existing one? We can't just pass it where we need it?

Reply:
no i dont think so either. we should definitely pass it in. do you know how i can access the `config.migrations.directory` from the knex instance returned on line 204?

```js
      const instance = initKnex(env, opts);
```

i wasnt able to figure that out myself before which resulted in this little surgery 😂 . if i can access it from the instance i can remove this duplication